src/query/core/buildMiddleware/queryLifecycle.ts

```diff
@@ -40,11 +40,14 @@ export const build: SubMiddlewareBuilder = ({
           originalArgs
         )
 
+        const extra = mwApi.dispatch((_, __, extra) => extra)
```

Comment:
🤣

Reply:
Genius, isn't it? :D

I was like "the old API had it, so we mayybe need it here too?" and then "how on earth do I get that?". Yup.